README.md

```diff
@@ -14,42 +14,71 @@ Bootstrap a [lxd controller](https://juju.is/docs/olm/lxd#heading--create-a-cont
 juju add-model postgresql
 ```
 
-To deploy this charm using Juju 2.9.0 or later, run:
+### Basic Usage
+To deploy a single unit of PostgreSQL using its default configuration.
 
 ```shell
-charmcraft pack
-juju deploy ./postgresql_ubuntu-20.04-amd64.charm
+juju deploy postgresql --channel edge
 ```
 
-Note: the above model must exist outside of a k8s environment (you could bootstrap an lxd environment).
-
-To confirm the deployment, you can run:
+It is customary to use PostgreSQL with replication. Hence usually more than one unit (preferably an odd number to prohibit a "split-brain" scenario) is deployed. To deploy PostgreSQL with multiple replicas, specify the number of desired units with the `-n` option.
 
 ```shell
-juju status
+juju deploy postgresql --channel edge -n <number_of_units>
 ```
 
-Once PostgreSQL starts up, it will be running on the default port (5432).
+To retrieve primary replica one can use the action `get-primary` on any of the units running PostgreSQL.
+```shell
+juju run-action postgresql/<unit_number> get-primary --wait
+```
```

Comment:
might want to add the disclaimer like you did [here](https://github.com/canonical/postgresql-k8s-operator/blob/62c636f7a957bfe234749b98df7516691959cbef/README.md?plain=1#L29)

Reply:
Yeah. In this case this feature is not implemented yet (only in the k8s charm). But I added it as an [issue](https://github.com/canonical/postgresql-operator/issues/22) in this repository. Thanks for checking this, Mia!